src/borg/archiver.py

```diff
@@ -2164,7 +2176,7 @@ def do_break_lock(self, args, repository):
             $ borg create -e '/home/*/junk' backup /
 
             # Exclude the contents of '/home/user/cache' but not the directory itself:
-            $ borg create -e /home/user/cache/ backup /
+            $ borg create -e home/user/cache/ backup /
```

Comment:
hmm, this (`/home/...`) never matched before your changes, right?

Reply:
Before, in ``create`` it matched only as `/home`.

Reply:
And: Before, in ``extract`` it matched only as `home`.

Reply:
uh, right.